src/Native/System.Native/pal_runtimeinformation.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_types.h"
+#include "pal_utilities.h"
+#include <sys/utsname.h>
+
+extern "C" int32_t UName(char* machine, int32_t capacity)
```

Comment:
If you're going to return a subset of what uname returns, call the shim function something more specific.